CombiningGame/Managers/GameManager.cs

```diff
@@ -7,7 +7,7 @@ namespace CombiningGame.Managers
 {
     public class GameManager
     {
-        private readonly TwoRandomOneToTenNumbersGenerator NumbersGenerator = new TwoRandomOneToTenNumbersGenerator();
+        private readonly TwoRandomOneToTenNumbersGenerator NumbersGenerator = new TwoRandomOneToTenNumbersGenerator(1, 10);
```

Comment:
You can name object of this class as "TwoRandomOneToTenNumbersGenerator", but not the class as self.